Gemfile.lock

```diff
@@ -109,19 +109,19 @@ GEM
     rack (1.5.5)
     rack-test (0.6.3)
       rack (>= 1.0)
-    rails (4.1.0)
-      actionmailer (= 4.1.0)
-      actionpack (= 4.1.0)
-      actionview (= 4.1.0)
-      activemodel (= 4.1.0)
-      activerecord (= 4.1.0)
-      activesupport (= 4.1.0)
+    rails (4.1.1)
```

Comment:
ERROR: [Medium][Mass Assignment] create_with is vulnerable to strong params bypass. Upgrade to Rails 4.1.5 or patch
https://groups.google.com/d/msg/rubyonrails-security/M4chq5Sb540/CC1Fh0Y_NWwJ